javascript/ql/test/library-tests/frameworks/SQL/SqlString.expected

```diff
@@ -41,7 +41,11 @@
 | sequelize-types.ts:7:24:7:35 | 'SELECT 123' |
 | sequelize.js:8:17:8:118 | 'SELECT ... Y name' |
 | sequelizeImport.js:3:17:3:118 | 'SELECT ... Y name' |
+| spanner2.js:4:19:4:32 | "not SQL code" |
```

Comment:
The test recognizes this "not SQL code"-string as an SQL-string?  

(Same as few lines below).

Reply:
That's right, I deliberately loosened the model a bit to not distinguish between endpoint that expect `string`, `{sql: string}`, or both. That tends to work out better IMO. I've updated the test to clarify on this.